packages/xdg_base.rb

```diff
@@ -3,12 +3,25 @@
 class Xdg_base < Package
   description 'XDG Base Directory Specification Configuration'
   homepage 'https://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html'
-  version '0.7-7'
+  version '0.7-7-1'
   license 'GPL-3+'
   compatibility 'all'
   source_url 'file:///dev/null'
   source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
 
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/xdg_base-0.7-7-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/xdg_base-0.7-7-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/xdg_base-0.7-7-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/xdg_base-0.7-7-1-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '2ec80e4af5d9791083b8004542b4fda18160a8773ef42a7390ec3018927c37ea',
+     armv7l: '2ec80e4af5d9791083b8004542b4fda18160a8773ef42a7390ec3018927c37ea',
+       i686: '34df15da8d205ac30a6a439829851502676e9f3484d1c823cdffb65eeba91327',
+     x86_64: '6e8adac27ccf434f6e4c227487cc6d0ab262093c34095170ec351537a003981b'
+  })
```

Comment:
We need binaries for one file? LoL

Reply:
It makes the install throw fewer error messages, so it looks cleaner on install...

Reply:
We need a crew alternative to /dev/null for the source url...